lib/Comment/CommentBody.js

```diff
@@ -1,51 +1,67 @@
 import React, { Fragment } from 'react';
 import { arrayOf, bool, func, node, shape } from 'prop-types';
 import Linkify from 'linkifyjs/react';
-import CommentForm from '../Conversation/CommentForm';
 import Person from '../Person';
+import { Comment } from './Comment';
 
 export function CommentBody({
   children,
   isEditing,
   comment,
   users,
-  onEditChange,
   onEditCancel,
-  onEditSubmit
+  onEditSubmit,
+  showFullText,
+  isDeleting
 }) {
+  const highlightMentions = () => {
+    const pattern = /(\B@\w+)+/gi;
+    const strArr = comment.body.split(pattern);
+    return strArr.map(subStr => {
+      if (subStr.match(pattern)) {
+        const username = subStr.substr(1);
+        const matches = users.filter(user => user.display === username);
+        if (matches.length) {
+          return (
+            <b key={subStr} title={matches[0].name}>
+              {subStr}
+            </b>
+          );
+        }
+      }
+      return subStr;
+    });
+  };
```

Comment:
@AmeeMorris had mentioned there was no mentions highlighting. It's back! I had taken it out thinking we'd do it in app, but I think it makes sense here